### DISCUSSION TITLE
Github Account Suspended

### DISCUSSION THREAD

**Discussion by WilliamHawkingStephen** [community/community#133643] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by queenofcorgis** [comment, marked as answer]
Hi @WilliamHawkingStephen, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

Please note:  contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.
